src/components/App.js

```diff
@@ -254,7 +254,8 @@ export default class App extends Component {
     return this.moveInHistory(1)
   }
 
-  handleDataChange = evt => {
+  handleDataChange = (evt, overwriteLastHistory = false) => {
```

Comment:
Why do we need `overwriteLastHistory`?

Reply:
Moving a node creates a entry in the history, and merging it would create another one. If I wouldn't overwrite the last history entry (the move) with the move and the merge, one could click undo and would have two not-merged nodes on the same position.

Reply:
I thought I fixed the undo/redo by doing so, but apparently I broke it again :/

Reply:
I added the true in the wrong spot. Weird that I didn't catch this before